Src/zipkin4net.middleware.owin/Src/Extensions/OwinExtensions.cs

```diff
@@ -7,10 +7,12 @@ namespace Owin
 {
     public static class OwinExtensions
     {
-        public static IAppBuilder UseZipkinTracer(this IAppBuilder appBuilder, string serviceName, IExtractor<IHeaderDictionary> traceExtractor, Func<IOwinContext, string> getRpc = null)
-            => appBuilder.Use<ZipkinMiddleware>(serviceName, traceExtractor, getRpc);
+        public static IAppBuilder UseZipkinTracer(this IAppBuilder appBuilder, string serviceName, IExtractor<IHeaderDictionary> traceExtractor, Func<IOwinContext, string> getRpc = null,
+            Func<PathString, bool> routeFilter = null)
+            => appBuilder.Use<ZipkinMiddleware>(serviceName, traceExtractor, getRpc, routeFilter);
```

Comment:
@bstauff Can we update the line limit to at most `120` so that it will be easier to view all the arguments? No need to scroll.

![image](https://user-images.githubusercontent.com/10608964/52461312-dc7e9600-2bb1-11e9-96d8-670d530b3fa3.png)

Please apply it on the method below too.

Reply:
done in eb9be3d0993e1bbaa3c429581d0ea72045fa0636